source/guides/clear/telemetrics.rst

```diff
@@ -163,23 +167,25 @@ Enable or disable telemetry
    During the initial installation of |CL|, you are requested to join the
    stability enhancement program and allow |CL| to collect anonymous reports to
    improve system stability. If you choose not to join this program, then the
-   telemetry software bundle is not added to your system. Choosing to join will
-   automatically enable telemetry on your system after installation is
-   complete.
+   telemetry software bundle is not added to your system. If you do choose to
+   join the program, the installer will automatically enable telemetry on your
+   system by installing the telemetrics bundle, creating the file
+   :file:`/etc/telemetrics/opt-in`, and enabling the telemtrics systemd
+   services to run after installation is complete and the system is restarted.
 
 #. Enabling after install:
 
-   To start telemetry on your system, run the following command:
+   To install telemetry on your system, run the following commands:
 
    .. code-block:: bash
 
+      sudo swupd bundle-add telemetrics
+      sudo telemctl opt-in
       sudo telemctl start
 
-   This enables and starts the :command:`telemprobd` and :command:`telempostd`
-   daemons. Your system will begin to send telemetry data to the server defined
-   in the file :file:`/etc/telemetrics/telemetrics.conf`. If this file does not
-   exist, the :command:`telemprobd` and :command:`telempostd` daemons will use
-   the file :file:`/usr/share/defaults/telemetrics/telemetrics.conf`.
+   This installs the necassary software, enables telemetry by creating the file
```

Comment:
That's because there are no records stored locally.  The --v just adds the header but we can add it if you want, doesn't hurt anything

sudo telemctl is-active will tell you if the daemons are running or not.

Reply:
the config file needs to be in /etc/telemetrics/telemetrics.conf.  When you opt-in the system will create an empty file in /etc/telemetrics/opt-in

Reply:
I found this when I was testing: sudo telemctl is-active
Why not include it here? 
As a user, it's nice to see confirmation.